### PR TITLE
Docs(web-react): Add Header demo #DS-898

### PR DESCRIPTION
**Pull request by pavelklibani**
## Description

Header demo in web-react is now same as demo in web and web-twig

### Issue reference

[Demo: Header má ve všech technologiích stejná dema](https://jira.lmc.cz/browse/DS-898)

### CODE REVIEW

File: packages/web-react/src/components/Header/demo/HeaderExperimentalRTL.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Checkbox } from '../../Checkbox';
+
+const HeaderExperimentalRTL = () => {
+  const toggleRtl = () => {
+    const htmlElement = document.querySelector('html');
+
+    if (htmlElement?.hasAttribute('dir')) {
+      document.querySelector('html')?.removeAttribute('dir');
```
Comment (literat) [inline packages/web-react/src/components/Header/demo/HeaderExperimentalRTL.tsx:9]: You can use the `htmlElement` variable here, can't you? And the same on the line 14.

File: packages/web-react/src/components/Header/demo/HeaderInvertedWithActions.tsx [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
-const Story: ComponentStory<typeof Header> = () => {
+const HeaderInvertedWithActions = () => {
   const [isOpen, setOpen] = useState(false);
 
   const handleOpen = () => setOpen(true);
   const handleClose = () => setOpen(false);
 
   return (
     <>
+      <p>Using the Offcanvas JS plugin.</p>
```
Comment (literat) [inline packages/web-react/src/components/Header/demo/HeaderInvertedWithActions.tsx:28]: This is React, it does not need any JS plugin.

Comment (literat) [reply packages/web-react/src/components/Header/demo/HeaderInvertedWithActions.tsx:28]: @adamkudrna @crishpeen I think we should remove these subheadings also from web and web-twig.

**Review by literat — CHANGES_REQUESTED** [review]
Although it is looking good to me and the demos are the same, we need to solve the problem with subheadings that describe the usage of the JS plugin. That, in the context of the React, does not seem correct.

File: packages/web-react/src/components/Header/demo/HeaderExperimentalRTL.tsx (hunk 1 above)
Comment (pavelklibani) [reply packages/web-react/src/components/Header/demo/HeaderExperimentalRTL.tsx]: you are right, fixed

File: packages/web-react/src/components/Header/demo/HeaderExperimentalRTL.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { Checkbox } from '../../Checkbox';
+
+const HeaderExperimentalRTL = () => {
+  const toggleRtl = () => {
+    const htmlElement = document.querySelector('html');
+
+    if (htmlElement?.hasAttribute('dir')) {
+      htmlElement?.removeAttribute('dir');
+
+      return;
```
[the remark this answers is not in the document]
Comment (pavelklibani) [reply packages/web-react/src/components/Header/demo/HeaderExperimentalRTL.tsx:11]: done

File: packages/web-react/src/components/Header/demo/HeaderInvertedWithActions.tsx (hunk 2 above)
Comment (literat) [reply packages/web-react/src/components/Header/demo/HeaderInvertedWithActions.tsx]: We agreed that we remove this subheading in all demos. So please remove this one. Thanks.

Comment (pavelklibani) [reply packages/web-react/src/components/Header/demo/HeaderInvertedWithActions.tsx]: removed

**Review by literat — APPROVED** [review]
LGTM 👍

**Review by crishpeen — APPROVED** [review]

File: packages/web-twig/src/Resources/components/Header/Header.stories.twig
[the remark this answers is not in the document]
Comment (pavelklibani) [reply packages/web-twig/src/Resources/components/Header/Header.stories.twig:1]: rebased and fix 😁

**Pull request merged**